### PR TITLE
fix(types): clean up `canonical-json` types in ambient.d.ts

### PR DESCRIPTION
**Pull request by lewisdaly**

### CODE REVIEW

**Review by kleyow — APPROVED** [review]

File: ambient.d.ts [hunk 1]
```diff
@@ -563,5 +563,6 @@ declare module '@hapi/good'
 declare module 'hapi-openapi'
 declare module 'blipp'
 declare module 'canonical-json' {
-  export default function stringify(value: any, replacer ?: ((this: any, key: string, value: any) => any) | undefined, space ?: string | number | undefined): string
+  export type ReplacerFunc = ((this: unknown, key: unknown, value: unknown) => unknown)
+  export default function stringify(value: any, replacer?: ReplacerFunc, space?: string | number): string
```
Comment (eoln) [inline ambient.d.ts:567]:
```suggestion
  export default function stringify(value: unknown, replacer?: ReplacerFunc, space?: string | number): string
```

**Review by eoln — CHANGES_REQUESTED** [review]
small change: better use unknown instead of any

File: ambient.d.ts (hunk 1 above)
Comment (lewisdaly) [reply ambient.d.ts:567]: oops! missed that one

**Review by eoln — APPROVED** [review]
lgtm

**Pull request merged**